.kotlin_alpha/usecases/first_android_app/Readme.md

```diff
@@ -0,0 +1,423 @@
+# Creating your first Native Android application using the AWS SDK for Kotlin
+
+## Purpose
+You can develop a native Android application that lets application users submit mobile data to an Amazon DynamoDB table by using the AWS SDK for Kotlin. In addition to using Amazon DynamoDB Kotlin API, this Android application also uses the Amazon Simple Notification Service (Amazon SNS) Kotlin API. For example, this Android application uses the **aws.sdk.kotlin.services.dynamodb.DynamoDbClient** to store data within a DynamoDB table. After the DynamoDB table is updated, the application uses the Kotlin Amazon SNS API to send a text message to notify a user. This application also uses the Android API to build the application logic. For more information, see [Android API reference](https://developer.android.com/reference.html).
+
+
+The Amazon DynamoDB Kotlin API lets you perform the following tasks from within an Android project:
+
+* Access your tables
+* Perform various create, read, update, and delete (CRUD) operations
+* Execute queries
+
+The following shows the application you'll create.
+
+![AWS Blog Application](images/app0.png)
+
+When you choose **Submit**, the data is persisted into an Amazon DynamoDB table named **Android**, as shown in the following illustration.
+
+![AWS Blog Application](images/table1.png)
+
+After the table is updated with a new item, a text message is sent to notify a mobile user who monitors the submissions.
+
+![AWS Blog Application](images/text1.png)
+
+This tutorial guides you through creating a sample Native Android application that invokes Amazon DynamoDB and Amazon Simple Notification Service (SNS) by using the AWS SDK for Kotlin. 
+
+#### Topics
+
++ Prerequisites
++ Create an Android project named **AWSAndroid**
++ Add the dependencies to your Andorid project
++ Create the layout XML file for your Android project
++ Create the Kotlin classes for your Android project
++ Run the Android application
+
+## Prerequisites
+
+To complete the tutorial, you need the following:
+
++ An AWS account
++ An IDE that lets you build Android projects (this example uses IntelliJ and the Android plugin. See [Android Studio Plugin Development](https://plugins.jetbrains.com/docs/intellij/android-studio.html) )
++ Java 1.8 SDK
++ Gradle 6.8 or higher
++ Min API version is 24
```

Comment:
I think you mean minimum Kotlin SDK API version is 24? Maybe better to be explicit.